phoenicis-javafx/src/main/java/org/phoenicis/javafx/settings/JavaFxSetting.java

```diff
@@ -20,8 +20,10 @@
 
 public enum JavaFxSetting {
     THEME("application.theme"), SCALE("application.scale"), VIEW_SOURCE("application.viewsource"), WINDOW_HEIGHT(
-            "application.windowHeight"), WINDOW_WIDTH(
-                    "application.windowWidth"), WINDOW_MAXIMIZED("application.windowMaximized");
+            "application.windowHeight"), WINDOW_WIDTH("application.windowWidth"), WINDOW_MAXIMIZED(
```

Comment:
I have tried that several times but the formatter does not respect manual line breaks. If you know which setting we have to change so that this works, let me know (it respects it for functions).